week04/index.html

```diff
@@ -0,0 +1,163 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="style.css">
+    
+    <title>가위바위보</title>
+</head>
+<body>
+    <div class="top_end"></div>
+    <div class="background">
+        <div class="top">
+            <div class="score">
+                <div id="humanscore">0</div>
+                <div id="vs"> : </div>
+                <div id="computerscore">0</div>
+            </div>
+            <div class="text">버튼을 눌러 시작하세요!</div>
+        </div>
+        
+        <div class="middle">
+            <div id="human_select"><img id="img" src="./img/paper.png"/></div>
+            <div id="robot_select"><img id="robotimg" src="./img/paper.png"/></div>
+        </div>
+        <div class="bottom">
+            <button onclick="scissor_button()" id="img_toScissor">가위</button>
+            <button onclick="rock_button()" id="img_toRrock">바위</button>
+            <button onclick="paper_button()" id="img_toPaper">보</button>
+        </div> 
+        <button id="reset">초기화</button>
+    </div>
+
+    <script>
+        let humanScore  = document.getElementById("humanscore");
+        let computerScore = document.getElementById("computerscore");
```

Comment:
변수는 기본적으로 `const`로 선언하고, 아주 특별한 경우에만 `let`으로 선언한다고 생각해야 합니다. 특히 `document.getElementById()`와 같이 요소를 불러와 저장하는 변수의 경우, 변수의 값이 변경될 일이 없으므로 `const`로 선언해야 합니다.
```suggestion
        const humanScore  = document.getElementById("humanscore");
        const computerScore = document.getElementById("computerscore");
```